yt/yt/client/api/operation_client.h

```diff
@@ -158,6 +158,10 @@ struct TListOperationsOptions
 
     TDuration ArchiveFetchingTimeout = TDuration::Seconds(3);
 
+    // These parameters cannot be set from list_operations command, they must be computed.
```

Comment:
I don't this that it is a good idea. This API can be used directly.

Reply:
It is better to pass it directly to corresponding methods and classes, or maybe introduce `struct TListOperationContext` and pass it